tests/test_job.py

```diff
@@ -136,7 +138,8 @@ def test_submit_job_url_with_success(self, mock_session, make_mock_response):
                 'custom_vocabularies': CUSTOM_VOCAB,
                 'filter_profanity': True,
                 'remove_disfluencies': True,
-                'delete_after_seconds': 0
+                'delete_after_seconds': 0,
+                'language' = LANG
```

Comment:
```suggestion
                'language': LANGUAGE
```

Reply:
I noticed this when I checked the CI build of your commit: https://travis-ci.org/github/revdotcom/revai-python-sdk/jobs/751115476